extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -78,8 +78,21 @@ function isValidOffsetArray(ary) {
       typeof ary[1] === 'number';
 }
 
+/** @type {string} */
 const METADATA_STRING = '<script type="application/json" amp-ad-metadata>';
 
+// TODO(tdrl): Temporary, while we're verifying whether SafeFrame is an
+// acceptable solution to the 'Safari on iOS doesn't fetch iframe src from
+// cache' issue.  See https://github.com/ampproject/amphtml/issues/5614
+/** @type {string} */
+const SAFEFRAME_VERSION = '1-0-4';
+/** @type {string} */
+const SAFEFRAME_IMPL_PATH =
+    'https://tpc.googlesyndication.com/safeframe/' + SAFEFRAME_VERSION +
```

Comment:
I tried it that way at first, but when I do the whole URL as a single string, it runs over line length limit.  Given that I have to split it somewhere anyway, I figured there was no readability advantage to interpolation vs. concatenation.  Can change if you feel strongly.